contrib/pg_tde/src/access/pg_tde_tdemap.c

```diff
@@ -1506,12 +1500,6 @@ GetSMGRRelationKey(RelFileLocatorBackend rel)
 		return GetRelationKey(rel.locator, TDE_KEY_TYPE_SMGR, true);
 }
 
-InternalKey *
-GetHeapBaiscRelationKey(RelFileLocator rel)
```

Comment:
Due to a typo in the function name you missed removing the prototype in the header file. :)